### DISCUSSION TITLE
XdripJH main graph screen shows unreadeable data

### DISCUSSION THREAD

**Discussion by hofno** [jakobheyman/xdripJH#4] [Q&A] [answered]
Hi! Im having terrible time trying to decode the readings on the graph (main) screen of the xdrip. Basically, everytime I start the app it shows me a cutout of the whole graph that is so detailed that it is completely unuseable in terms of informational value. I can zoom out of it but when I restart the app or just leave it and come back to it, it defaults back to ultrazoom. I am attaching screenshots for the reference. Tried fixing it in the settings but to vein. also used xdrip+ on a different phone till recently and had no such issues at all.

Techincal data: 
im using latest version of xdripJH  @jakobheyman (was using latest xdrip+ couple of days back with same results) connected to fresstyle libre 2 EU over bluetooth with oop2 algorythm app. my phone is redmi 12C MIUI Global 14.0.7 on Android 13

![xdripstatus](https://github.com/jakobheyman/xdripJH/assets/24904174/5e784dfd-0d16-473f-91b6-24260e9fa287)
![xdrip screen2](https://github.com/jakobheyman/xdripJH/assets/24904174/f59cb35f-d008-470f-a865-05924f8036a6)
![xdrip screen1](https://github.com/jakobheyman/xdripJH/assets/24904174/c0819680-f286-47e4-b031-f7a94aa78d71)

**Comment by jakobheyman** [comment]
The range of values shown can be controlled by the parameters "Graph min Y" and "Graph max" which give the min and max blood glucose values used for the Y axis as long as the blood glucose is within that range. These parameters can be found under Settings > xDrip+ Display Settings > Graph Settings.

**Reply by hofno** [reply]
Thank you! i played with those parameters but perhaps im lacking the scale.. what values would you suggest?

**Reply by jakobheyman** [reply, marked as answer]
If you use mmol/l as unit you can perhaps try with Graph min Y at 2 and Graph max Y at 12. With those values you should always get a range of 10 mmol/l between the max and min value of the zoom bar. The zoom bar will move up and down with the blood glucose based on the values of the last hour so that if you have a recent highest value of 16 mmol/l the Y axis will range from 6 to 16 mmol/l. If you use mg/dl as units you will have to use other (higher) values. It should be possible to just change "Graph min Y" and "Graph max Y" as often as you wish to try to get a range that you like.

**Comment by hofno** [comment]
thanks, that solved the problem. may i suggest that if this is something bundled as default in xdripJH it is changed in next version to more human-readeable value by default. thanks

**Reply by jakobheyman** [reply]
Good idea!
I believe the problem was that the default values were set to 2 and 10.4 since I have always used mmol/l as unit. Probably those values were divided by 18.0156 if using mmol/l and therefore the Y range got much too small. In the latest version that I just uploaded I multiplied the default values with 18.0156. Hopefully this will fix this issue.
